src/modules/new-request-form/translations/en-us.yml

```
@@ -292,3 +292,9 @@ parts:
       title: "[A11Y] Aria label value for the close modal button"
       value: "Close modal"
       screenshot: "https://drive.google.com/file/d/11qwmad1gcWHjtJ6LhyRxk1d9BlREjxwr/view?usp=sharing"
+  - translation:
+      key: "new-request-form.description.aria_label"
+      title: "[A11Y] Aria label value for the description field"
+      value: "Description editing area"
+      screenshot: "https://drive.google.com/file/d/1wDm8jmFjlUTkJlPq_OfHeJRCTanf5BOi/view?usp=drive_link"
```

It would be great if we could also translate the instructions: `Press ⌥0 for help`. 

I'm wondering if we'd also need to do some localization. Doing some quick Googling, it sounds like the Option key is similar to the [AltGr key](https://en.wikipedia.org/wiki/AltGr_key) on some European keyboards. Does pressing AltGr + 0 achieve the same result as pressing Option + 0? If so, we might want to consider having the description text say that. 🤔 Let's check in with @olivia-tsao & co., on that.

```suggestion
  - translation:
      key: "new-request-form.description.aria_label"
      title: "[A11Y] Aria label value for the description field"
      value: "Description editing area"
      screenshot: "https://drive.google.com/file/d/1wDm8jmFjlUTkJlPq_OfHeJRCTanf5BOi/view?usp=drive_link"
  - translation:
      key: "new-request-form.description.aria_describedby"
      title: "[A11Y] Aria-describedby value for the description field."
      value: "Press ⌥0 for help"
      screenshot: "https://drive.google.com/file/d/..."    
```

---

@gosiexon-zen Agree that shortcuts can be different for some langs depending on the keys used/ available in the language. In such cases, extracting the keys and asking translators to change according to the language can be helpful.